### DISCUSSION TITLE
MX record & sending emails from other instances than SH SimpleLogin

### DISCUSSION THREAD

**Discussion by melbv** [simple-login/app#612] [Self-hosting question] [answered]
HI all,
Having just set up a self-hosted instance of SimpleLogin on a custom domain, which works like a breeze and I love, I am trying to wrap my head around interaction btw domain MX/SL/sending emails for other sub-domains of the same domain. 
This may be a newbie question and i apologise in advance, but as the domain's MX record points to the self-hosted SimpleLogin instance, I seem to have trouble sending emails from other sub-domains on the same domain. I hence have one question:
What is the appropriate smtp server (or otherwise email sending protocol) to call to send mail (admin mail mostly) from other sub-domains of the same domain than the SL set-up?
Many thanks in advance for your help.
All the best

**Comment by nguyenkims** [comment]
Hey by "sending emails from other sub-domains", do you mean you want to use the same Postfix server (that's part of SimpleLogin self-hosted code) to send emails from a subdomain? SimpleLogin can also handle subdomain so it's maybe better to add your subdomain into SimpleLogin and manage it from there?

**Comment by melbv** [comment]
Hi Son
I mean indeed leveraging on Postfix set up for SimpleLogin.
In short, I chose to plug directly SimpleLogin on my domain, rather than subdomain, to avoid lengthy radicals for my aliases. Yet, in this configuration, i would require other services under the domain umbrella to be dialoguing with SL's Postfix to send emails... It seems to me to be cleaner as i can create any email address relating to that domain as an alias but am falling short of the Postfix (other subdomain service)/Postfix (SH SL) dialogue.
Hope this makes sense

**Comment by melbv** [comment, marked as answer]
Hi Son
I finally found a solution on this one by looking around. 
Please find below an article explaining how to amend SL's template for Postfix's main.cf & master.cf.
https://serverfault.com/questions/768313/postfix-mail-server-refuses-connections-from-outside-mail-servers/768376

And below a way to parameter another instance of Postfix to push emails through an external SMTP (example here for Debian/Ubuntu)
https://www.linode.com/docs/guides/postfix-smtp-debian7/

I just tested and my configs seem to work.

**Reply by nguyenkims** [reply]
Thanks and glad that you found the solution!
